src/ProjectService.Business/Commands/Image/CreateImageCommand.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace LT.DigitalOffice.ProjectService.Business.Commands.Task
+{
+    public class CreateImageCommand : ICreateImageCommand
+    {
+        private readonly IImageRepository _repository;
+        private readonly IRequestClient<ICreateImagesRequest> _rcImages;
+        private readonly ILogger<CreateImageCommand> _logger;
+        private readonly IAccessValidator _accessValidator;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly IDbProjectImageMapper _dbProjectImageMapper;
+        private readonly ICreateImageValidator _validator;
+
+        private List<Guid> CreateImage(List<ImageContent> context, Guid userId, List<string> errors)
+        {
+            List<CreateImageData> images = context
+                .Select(x => new CreateImageData(x.Name, x.Content, x.Extension, userId))
+                .ToList();
+
+            string errorMessage = "Can not create images. Please try again later.";
+            const string logMessage = "Errors while creating images for user id {userId}. Errors: {Errors}";
+
+            try
+            {
+                Response<IOperationResult<ICreateImagesResponse>> brokerResponse = _rcImages.GetResponse<IOperationResult<ICreateImagesResponse>>(
+                   ICreateImagesRequest.CreateObj(images, ImageSource.Project)).Result;
+
+                if (brokerResponse.Message.IsSuccess && brokerResponse.Message.Body != null)
+                {
+                    return brokerResponse.Message.Body.ImagesIds;
+                }
+
+                _logger.LogWarning(
+                    logMessage,
+                    string.Join('\n', userId),
+                    string.Join('\n', brokerResponse.Message.Errors));
+            }
+            catch (Exception exc)
+            {
+                _logger.LogError(exc, logMessage);
+            }
+
+            errors.Add(errorMessage);
+
+            return null;
+        }
+
+        public CreateImageCommand(
+            IImageRepository repository,
+            IRequestClient<ICreateImagesRequest> rcImages,
+            ILogger<CreateImageCommand> logger,
+            IAccessValidator accessValidator,
+            IHttpContextAccessor httpContextAccessor,
+            IDbProjectImageMapper dbProjectImageMapper,
+            ICreateImageValidator validator)
+        {
+            _repository = repository;
+            _rcImages = rcImages;
+            _logger = logger;
+            _accessValidator = accessValidator;
+            _httpContextAccessor = httpContextAccessor;
+            _dbProjectImageMapper = dbProjectImageMapper;
+            _validator = validator;
+        }
+
+        public OperationResultResponse<List<Guid>> Execute(CreateImageRequest request)
+        {
+            OperationResultResponse<List<Guid>> response = new();
+            List<string> errors = new();
```

Comment:
create errors list after 102 string because you can dont pass first check and you dont need it object